conda-recipe/build.sh

```diff
@@ -22,31 +25,34 @@ DPCPP_ROOT=${ONEAPI_ROOT}/compiler/latest/linux/
 
 cmake                                                       \
     -DCMAKE_BUILD_TYPE=Release                              \
-    -DCMAKE_INSTALL_PREFIX=${PREFIX}                        \
-    -DCMAKE_PREFIX_PATH=${PREFIX}                           \
+    -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX}                \
+    -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX}                   \
     -DDPCPP_ROOT=${DPCPP_ROOT}                              \
     -DPYTHON_INCLUDE_DIR=${PYTHON_INC}                      \
     -DNUMPY_INCLUDE_DIR=${NUMPY_INC}                        \
     ../backends
 
 make -j 4 && make install
 
-cd ..
+popd
+cp install/lib/*.so dpctl/
+
+mkdir -p dpctl/include
+cp -r backends/include/* dpctl/include
 
 # required by dpctl.opencl_core
-export DPPL_OPENCL_INTERFACE_LIBDIR=${PREFIX}
-export DPPL_OPENCL_INTERFACE_INCLDIR=${PREFIX}/include
-export OpenCL_LIBDIR=${DPCPP_ROOT}/lib
+export DPPL_OPENCL_INTERFACE_LIBDIR=dpctl
+export DPPL_OPENCL_INTERFACE_INCLDIR=dpctl/include
+export OpenCL_LIBDIR=${DPCPP_ROOT}/libw
```

Comment:
libw -> lib? typo?

Reply:
I think `lib` is correct. It was based on this https://github.com/IntelPython/dpctl/pull/69/commits/24182369db7a69b4e2ef9c21c6e233dad52c453a.